internal/api/user_devices.go

```diff
@@ -111,6 +113,30 @@ func (s *userDeviceService) ListUserDevicesForUser(ctx context.Context, req *pb.
 	return &pb.ListUserDevicesForUserResponse{UserDevices: out}, nil
 }
 
+func (s *userDeviceService) ListDevicesForWalletAddress(ctx context.Context, req *pb.ListDevicesForWalletAddressRequest) (*pb.ListDevicesForWalletAddressResponse, error) {
```

Comment:
I wonder if we should fold this into the old endpoint. We left outer join `user_devices` with `vehicle_nfts` and do `user_devices.user_id = ? OR vehicle_nfts.owner_address = ?`?

Reply:
Or maybe a flag on the existing `ListUserDevicesForUser`? I'm not sure.